### PR TITLE
HW01: Self Introduce_Rui Tang

### PR DESCRIPTION
**Pull request by Ray-RuiTang**
Following the instruction, I cloned the README.md into my repositories. Then, I created a new project from version control and pasted the URL cloned from git.  When I edited README.md, I tried all elements listed on the homework website. I wrote down four sentences to introduce myself a little. The instruction of it is pretty straight forward. I found a successful markdown syntax will be highlighted, which could help me to check if it is worked. After finishing the biography, I committed the file and pushed it into my GitHub. Finally, by clicking the new pull request button, I shared my work into cfss-fa20. 

Overall, it was not very difficult to finish the assignment, though the committing part could be a bit confused. It is still necessary to click the push it after clicking the Commit button. I forgot about it the first time. The introduction of the workflow is detailed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
...
+
+## Self-introduction
+1. My name is **Rui Tang**. 
+2. I am a *MAPSS* student in _Psychology_.
+1. I am interested in studying individual differences. 
+3. I hope to learn some useful skills in this course!
+
+![Logo](https://cdn.pixabay.com/photo/2018/11/24/13/46/spongebob-3835614_960_720.jpg) 
+
+[My_Page](https://github.com/Ray-RuiTang)
```
Comment (ennissean) [inline README.md:11]: Excellent job adding this link, though it's maybe a bit redundant to link us back to your github account from the file we are viewing in github ;).

File: README.md [hunk 2]
```diff
@@ -1 +1,11 @@
 # hw01
+
+## Self-introduction
+1. My name is **Rui Tang**. 
+2. I am a *MAPSS* student in _Psychology_.
```
Comment (ennissean) [inline README.md:5]: Nice job with consistent italics to refer to programs after using bold for your name

File: README.md [hunk 3]
```diff
@@ -1 +1,11 @@
...
+1. I am interested in studying individual differences. 
+3. I hope to learn some useful skills in this course!
+
+![Logo](https://cdn.pixabay.com/photo/2018/11/24/13/46/spongebob-3835614_960_720.jpg)
```
Comment (ennissean) [inline README.md:9]: This picture is a bit big compared to the shorter 4 line list, and a little oddly chosen, but it is exceedingly cute.

File: README.md [hunk 4]
```diff
@@ -1 +1,11 @@
 # hw01
```
Comment (ennissean) [inline README.md:1]: Overall, great work! I feel like I got to know you a little and I want to know more.

**Review by ennissean — COMMENTED** [review]
Good job on this project, and I have your github link now to check out your other works throughout the rest of the quarter.

**Review by Donghyun-Kang-Soc — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      X        |            |
| **Coding strategy**      |           |       X       |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |           |    X          |            |
| **Reproducibility**         |    X       |              |            |

Remarks:
* You used headers, emphasis, an unordered list, and embedded a link and image. They hit the five aspects of the markdown. But please try to experiment with other elements as the course proceed when you submit other assignments.
* I expected to see a bit more of your biography and some combinations of different markdown aspects.

**Comment by maelissvillepontoux** [comment]
Hi Rui (sorry for the late peer review )
Good job on this assignment, you hit all the marks.  Also good job explaining everything you did in detail on the pull request. You could have perhaps toyed with using both a title & a subtitle!

**Review by maelissvillepontoux — COMMENTED** [review]
Hi Rui (sorry for the late peer review )
Good job on this assignment, you hit all the marks. Also good job explaining everything you did in detail on the pull request. You could have perhaps toyed with using both a title & a subtitle!

**Pull request closed without merging**